### ISSUE TITLE
[Label]: required * should be aria-hidden inside label

### ISSUE THREAD

**Issue by kasparsuvi1** [TEHIK-EE/tedi-design-system#245]
### Problem
When Label and FormLabel were added (#214) then the new Tedi component removed `aria-hidden="true"` from the required span.
It should be kept, so that it's removed from accessibilty tree. The * is only visual, accessibilty wise required is already read from input itself.
